pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java

```diff
@@ -99,10 +99,10 @@ public long getAvailableDispatchRateLimitOnByte() {
      */
     public boolean tryDispatchPermit(long msgPermits, long bytePermits) {
         boolean acquiredMsgPermit = msgPermits <= 0 || dispatchRateLimiterOnMessage == null
-        // acquiring permits must be < configured msg-rate;
+        // In dispatch mode, acquiring permits can be larger than configured msg-rate.
```

Comment:
dispatch mode represents what?

Reply:
> dispatch mode represents what? it is singleDispatcher and multiDispatcher, right?

Yes, I have removed these two java-doc because it is unnecessary.